testdata/dnn/darknet/cost.cfg

```diff
@@ -0,0 +1,10 @@
+[net]
+batch=1
+channels=3
+height=2
+width=4
+
+[avgpool]
+
+[cost]
+type=sse
```

Comment:
Let's add it to one of the existing tests

Reply:
Done !